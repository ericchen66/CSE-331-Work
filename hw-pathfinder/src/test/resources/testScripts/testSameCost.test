# Find a path with Dijkstra's algorithm when
# two paths between two nodes has the same cost

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D

AddEdge simpleGraph A C 5.0
AddEdge simpleGraph A D 5.0
AddEdge simpleGraph C B 5.0
AddEdge simpleGraph D B 5.0

FindPath simpleGraph A B
