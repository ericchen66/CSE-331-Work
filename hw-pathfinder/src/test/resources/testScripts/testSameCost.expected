# Find a path with Dijkstra's algorithm when
# two paths between two nodes has the same cost

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph

added edge 5.000 from A to C in simpleGraph
added edge 5.000 from A to D in simpleGraph
added edge 5.000 from C to B in simpleGraph
added edge 5.000 from D to B in simpleGraph

path from A to B:
A to C with weight 5.000
C to B with weight 5.000
total cost: 10.000
